src/core/laser_scan_grid_world.h

```diff
@@ -8,15 +14,27 @@
 #include "maps/grid_cell_strategy.h"
 #include "maps/grid_map.h"
 
+/**
+ * \brief Class deriver of World.
+ * Considers Map as Grid Map, and input data as Laser Scan. Relies on information from robot, updates a map
+ */
 class LaserScanGridWorld : public World<TransformedLaserScan, GridMap> {
 public: //types
   using MapType = GridMap;
   using Point = DiscretePoint2D;
 public: // methods
 
+  /**
+   * Constructor with parameters. Creates a world as a Map consisting of Grid Cells
+   * \param gcs Shared pointer on Cell
+   */
   LaserScanGridWorld(std::shared_ptr<GridCellStrategy> gcs) :
     _map(gcs->cell_factory()) {}
 
+  /**
+   * Gets pose of a Robot and calls handle_scan_point with some information about current location of robot
```

Comment:
Consider: "Updates the map cells according to given sensor data. Straightforward scan points projection is used."